### PR TITLE
Small recipe changes for net-snmp and ntp

### PR DESCRIPTION
**Pull request by ksorensen**

### CODE REVIEW

File: recipes/ntp/ntp.inc [hunk 1]
```diff
@@ -78,8 +78,11 @@ AUTO_PACKAGE_UTILS = "ntpd ntpdate ntp-wait ntp-keygen sntp ntpq ntpdc \
 RDEPENDS_${PN}-ntpd = "libm"
 RDEPENDS_${PN}-sntp = "libm"
 
-FILES_${PN}-ntpd = "${sysconfdir}/ntp.conf \
-    ${sysconfdir}/init.d/ntpd ${sysconfdir}/rc?.d/???ntpd"
+PACKAGES =+ "${PN}-config ${PN}-init"
```
Comment (kimrhh) [inline recipes/ntp/ntp.inc]: you should probably make some of the other packages pull in these new ones, at least PN

File: recipes/net-snmp/net-snmp.inc [hunk 2]
```diff
@@ -62,4 +73,5 @@ RDEPENDS_${PN}-client += "${AUTO_PACKAGE_UTILS_PROVIDES}"
 
 FILES_${PN}-mibs = "${datadir}/snmp/mibs"
 FILES_${PN}-client += "${datadir}/snmp"
-FILES_${PN}-server += "${sysconfdir} ${sbindir}/*"
+FILES_${PN}-server += "${sysconfdir}/init.d ${sbindir}/*"
+FILES_${PN}-conf += "${sysconfdir}/snmp"
```
Comment (kimrhh) [inline recipes/net-snmp/net-snmp.inc]: you should probably set up RDEPENDS to pull this in when someone just pulls in server or client

Comment (ksorensen) [inline recipes/net-snmp/net-snmp.inc]: ${PN}-server now pulls in ${PN}-conf

File: recipes/ntp/ntp.inc (hunk 1 above)
Comment (ksorensen) [inline recipes/ntp/ntp.inc]: ${PN}-ntpd rdepends on ${PN}-config and ${PN}-init

**Pull request merged**